File: despatch.WEB/app/src/swagger/models/image-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { UrlResponse } from './url-response';
import { ImageType } from './image-type';
export interface ImageResponse extends BaseEntityResponse{
  description?: string;
  deviceUrl?: UrlResponse;
  deviceUrlId?: string;
  isMain: boolean;
  name?: string;
  sourceUrl?: UrlResponse;
  sourseUrlId?: string;
  type: ImageType;
}

File: despatch.Domain/Models/Default/Response/ImageResponse.cs
using despatch.Domain.Enums;

namespace despatch.Domain.Models.Default.Response
{
    public class ImageResponse : BaseEntityResponse
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsMain { get; set; }
        public ImageType Type { get; set; }

        public string DeviceUrlId { get; set; }
        public string SourseUrlId { get; set; }

        public virtual UrlResponse DeviceUrl { get; set; }
        public virtual UrlResponse SourceUrl { get; set; }
    }
}
